libosmscout/src/osmscout/util/String.cpp

```diff
@@ -191,6 +192,27 @@ namespace osmscout {
     return "false";
   }
 
+  std::string NumberToString(long value, const Locale &locale)
```

Comment:
Problem is that C++ `std::locale` don't support utf-8 thousands separators. It support just one character: `std::numpunct<char>::thousands_sep`, so it may be `char` or `wchar_t`. So in case that you want to use non-breakable space as separator we would need to use wchar strings and then convert it to utf-8 every time... Number-to-string conversion with `osmscout::Locale` is used on two places now - elevation feature value and elevation profile chart. So was making sense to me put that method to string utilities...